lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -145,8 +145,9 @@ defmodule Kernel.ParallelCompiler do
         each_long_compilation: Keyword.get(options, :each_long_compilation, fn _file -> :ok end),
         each_module: Keyword.get(options, :each_module, fn _file, _module, _binary -> :ok end),
         beam_timestamp: Keyword.get(options, :beam_timestamp),
-        output: output,
         long_compilation_threshold: Keyword.get(options, :long_compilation_threshold, 15),
+        time_checker: Keyword.get(options, :time_checker, false),
```

Comment:
I would like to time everything, not only the checker. Also, because we may add other modes in the future, maybe we could have something like `--profile :time` instead?

Reply:
What else should be timed?

Reply:
Each compilation cycle as well.